internal/controllers/common/resource_definitions/resource_definitions.go

```diff
@@ -374,6 +376,14 @@ func NewCoreContainer(cr *operatorv1beta1.Cryostat, specs *ServiceSpecs, imageTa
 				Name:  "CRYOSTAT_AUTH_MANAGER",
 				Value: "io.cryostat.net.OpenShiftAuthManager",
 			},
+			{
+				Name:  "CRYOSTAT_OAUTH_CLIENT_ID",
+				Value: cr.Name,
+			},
+			{
+				Name:  "CRYOSTAT_OAUTH_ROLE",
+				Value: "cryostat-operator-cryostat",
```

Comment:
@ebaron I tried adding environment variables inside `openshiftEnvs := []corev1.EnvVar{...}` for cryostat-sample to access the names of the `serviceaccount` and `clusterRole`, but I don't see these env vars listed on OCP > Pods > cryostat-sample > Environment > Container: cryostat-sample. `env.getEnv("CRYOSTAT_OAUTH_ROLE")` also returns `null`. Any ideas?

Reply:
It looks okay to me, I'll try building and deploying it myself and let you know how it goes.

Reply:
Thanks, your build steps worked!